### PR TITLE
Remote query tests using JPQL

### PR DESCRIPTION
**Pull request by mgencur**

### CODE REVIEW

File: src/test/cs/Infinispan/HotRod/RemoteQueryTest.cs [hunk 1]
```diff
@@ -25,42 +34,495 @@ public void BeforeClass()
...
+            Assert.AreEqual(0, listOfUsers.Count);
+        }
+
+        [Test]
+        public void EqSentenceTest()
+        {
+            IRemoteCache<int, Account> accountCache = remoteManager.GetCache<int, Account>(NAMED_CACHE);
+
+            QueryRequest qr = new QueryRequest();
+            qr.JpqlString = "from sample_bank_account.Account a where a.description = \"John Doe's first bank account\"";
```
Comment (anistor) [inline src/test/cs/Infinispan/HotRod/RemoteQueryTest.cs]: You could use single quote instead of double quote, so it will not have to be escaped.

**Comment by mgencur** [comment]
Can you integrate this PR, please? Do I need to satisfy the comment from Adrian? It seems pretty minor.

**Comment by rigazilla** [comment]
@mgencur pull request #80 needs to be merged before this one. It contains all the updated to compile c# against the new version of the cpp-client. Can you review it?

**Comment by mgencur** [comment]
The PR #80 has been merged and this PR #79 is now rebased on master and ready for review.

**Comment by rigazilla** [comment]
merged thanks

**Pull request merged**